database/migrations/2024_11_26_011849_create_telemetry_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('telemetry', function (Blueprint $table) {
```

Comment:
I'm wondering if it may be cleaner to do a sort of `settings` table here which consists of a `KeyValue` setup (`key`, `value`) — no need for an `id` as `key` would be a string-based primary key (`$table->string('key')->primary()`).

That way we can just define a `SettingsEnum : string` with a case for `TELEMETRY_UUID = 'telemetry_uuid'` and read/write it based on that.

This probably saves having a dedicated table for `telemetry` which isn't reusable in any way. From a customisability perspective, other operators could then add their own custom `settings` leveraging this new table. I wouldn't say timestamps are required, so probably can just leave it as a simple `(key, value)` table.